docs/sql-data-sources-text.md

```diff
@@ -45,7 +45,7 @@ Data source options of text can be set via:
   *  `DataFrameWriter`
   *  `DataStreamReader`
   *  `DataStreamWriter`
-  *  `OPTIONS` clause at [CREATE TABLE USING DATA_SOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
+* `OPTIONS` clause at [CREATE TABLE USING DATA_SOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
```

Comment:
Adressed https://github.com/apache/spark/pull/32660#discussion_r639549599 here.

FYI, @HyukjinKwon

Reply:
Let's do this all in a separate PR because you'll have to fix other data source pages too (e.g., ORC and Parquet)

Reply:
Got it, I'll revert this change and do it in SPARK-35528 with other pages.